Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM openshift/origin-release:golang-1.21 AS build
+FROM golang:1.21 AS build
 WORKDIR /go/src
 COPY . /go/src
 RUN GIT_COMMIT=$(git rev-parse HEAD) && \
   CGO_ENABLED=0 GOOS=linux go build -a -mod=readonly \
   -ldflags "-X github.com/redhat-developer/gitops-backend/pkg/health.GitRevision=${GIT_COMMIT}" ./cmd/backend-http
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal
-WORKDIR /root/
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM gcr.io/distroless/static:nonroot
```

Comment:
For Go binaries built with `CGO_ENABLED=0` option , we could use `FROM scratch`. I assume there is no other other dependency required for this binary (like bash etc).